.github/ISSUE_TEMPLATE/🤔-submit-an-idea-for-a-template.md

```diff
@@ -8,20 +9,10 @@ assignees: nitya, savannahostrowski
 
 ---
 
-## Template Idea
+## 1. Template Description
 
-## 1. Suggested template name
-_A brief name to provide context_.
-
-## 2. Description of the application scenario/architecture
-_What is this the purpose of this template? Are there specific pieces of technology or a specific stack you're interested in?_.
-
-## 3. Architecture Diagram (`optional`)
-_An image of the architecture, if possible_
-
-## 4. Additional Tags (`optional`)
-_Don't see a relevant standard tag for your template? Request these tags be added, and then used for your template. These could be new languages, services, or application domains_.
+## 2. Azure Services (`optional`)
+_If there are specific azure services you'd like the template to use, please specify them here._
```

Comment:
I like the direction this is headed in. It allows for someone to granularly specify the services they might know of and want to use while also allowing them to submit an issue without any Azure specifics